index.js

```diff
@@ -106,8 +106,11 @@ module.exports = function() {
 
 		var preBlock = findPreviousSibling(highlight.parentElement, 'pre');
 		var codeBlock = preBlock.childNodes.item(0);
-
-		var total = codeBlock.innerHTML.split('\n').length - 1;
+		//Without trimming, additional characters can create new lines
+		//this happens when PrismJS is not applied (in testing)
+		//setting total to length - 1 makes the last not collapsed
+		codeBlock.textContent = codeBlock.textContent.trim();
```

Comment:
Is this the best place to do the trimming, or should it be done wherever `codeBlock.textContent` is initially created/set?

Reply:
Yeah, it could be trimmed initially in [bit-docs-prettify](https://github.com/bit-docs/bit-docs-prettify/blob/master/prettify.js#L43)

Reply:
@chasenlehara `bit-docs-prettify` is not invoked in testing, so I changed the fix to be the same as the needed [part](https://github.com/bit-docs/bit-docs-prettify/blob/master/prettify.js#L48-L50) to remove the newline at the end.